calendar-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -65,6 +65,9 @@
       <trans-unit id="MSC.eventPicker">
         <source>Events</source>
       </trans-unit>
+      <trans-unit id="MSC.eventContentPicker">
+        <source>Events</source>
```

Comment:
Is there no way to re-use `MSC.eventPicker` here?

Reply:
Not sure 🤷‍♂ it's a different picker, so it should have a different label. Then again, the tab/label actually represents the back end module, not the picker implementation…